### PR TITLE
updated challenge description

### PR DESCRIPTION
**Pull request by julietochu**

### CODE REVIEW

File: docs/description.md [hunk 1]
```diff
@@ -1,35 +1,36 @@
...
+5. Host can view the guest profile.
+6. Guest ca view the host profile.
+7. Host can delete occupied listing.
+8. Guest can search by city or name a location.
+9. Guest and host can chat.
+10. Guest can get the direction via a map.
+11. Host can accept or decline a request.
+12. Guest can view a specific listing.
+13. Guest can book a space.
+14. Guest can post gifs.
```
Comment (nataphilips) [inline docs/description.md]: Let's remove gifs? I'm not sure they need them in their chat.

File: docs/description.md [hunk 2]
```diff
@@ -81,18 +82,22 @@ NB:
...
+   - Host can view the guest profile.
+   - Guest can view the host profile.
+   - Host can delete occupied listing.
+   - Guest can search by city or name a location.
+   - Guest and host can chat.
+   - Guest can get the direction via a map.
+   - Host can accept or decline a request.
+   - Guest can view a specific listing.
+   - Guest can book a space.
+   - Guest can post gifs.
```
Comment (nataphilips) [inline docs/description.md]: Same here about gifs

**Review by nataphilips — CHANGES_REQUESTED** [review]

**Review by julietochu — COMMENTED** [review]
Updated

File: docs/description.md (hunk 2 above)
Comment (julietochu) [reply docs/description.md]: I agree, I have edited it.

File: docs/description.md (hunk 1 above)
Comment (julietochu) [reply docs/description.md]: Yeah, we will keep it simple.

**Pull request merged**